cdk/infrastructure/runtime/lambdas/snapshot/main.py

```diff
@@ -47,6 +47,11 @@ def sort_results_by_create_time(results):
     )
 
 
+def filter_results_by_db_instance_identifier(results, db_instance_identifier):
+    filtered_results = [result for result in results if result['DBInstanceIdentifier'] == db_instance_identifier]
```

Comment:
Everything will always have a `DBInstanceIdentifier`, right?

Reply:
Yes, that should be the case according to boto docs.